components/gif_picker/components/InfiniteScroll/index.ts

```diff
@@ -45,21 +38,28 @@ export default class InfiniteScroll extends PureComponent {
     render() {
         const {
             children,
-            element,
-            hasMore,
-            initialLoad, // eslint-disable-line no-unused-vars
+            element = 'div',
+            hasMore = false,
+            // eslint-disable-next-line @typescript-eslint/no-unused-vars
+            initialLoad = true,
             loader,
-            loadMore, // eslint-disable-line no-unused-vars
-            pageStart, // eslint-disable-line no-unused-vars
-            threshold, // eslint-disable-line no-unused-vars
-            useWindow, // eslint-disable-line no-unused-vars
-            isReverse, // eslint-disable-line no-unused-vars
-            scrollPosition, // eslint-disable-line no-unused-vars
-            containerHeight,
+            // eslint-disable-next-line @typescript-eslint/no-unused-vars
+            loadMore,
+            // eslint-disable-next-line @typescript-eslint/no-unused-vars
+            pageStart = 0,
+            // eslint-disable-next-line @typescript-eslint/no-unused-vars
+            threshold = 250,
+            // eslint-disable-next-line @typescript-eslint/no-unused-vars
+            useWindow = true,
+            // eslint-disable-next-line @typescript-eslint/no-unused-vars
+            isReverse = false,
+            // eslint-disable-next-line @typescript-eslint/no-unused-vars
+            scrollPosition = null,
```

Comment:
`scrollPosition` and `containerHeight` are optional, so their values are either a number or undefined, so they don't need a default value here.